v1/ast/term.go

```diff
@@ -614,9 +586,21 @@ func (null Null) String() string {
 // Boolean represents a boolean value defined by JSON.
 type Boolean bool
 
+// These are effectively constants, but casting to Value allocates,
+// so if that's an issue, these can be referenced whenever you need to
+// pass or return a Boolean as Value. Obviously, things will break in
+// all kinds of ways if you modify these.
+var (
+	BooleanTrueValue  Value = Boolean(true)
+	BooleanFalseValue Value = Boolean(false)
```

Comment:
```suggestion
	booleanTrueValue  Value = Boolean(true)
	booleanFalseValue Value = Boolean(false)
```

Why not do this? If it's not used from the outside. I just did a quick search, I might have missed some diff that's "too large" in GitHub's eyes 🙄

Reply:
Yay thanks